### ISSUE TITLE
Is "Get User Contest Records" working?

### ISSUE THREAD

**Issue by corey-lin** [JacobLinCool/LeetCode-Query#80]
Hi, 

thanks for creating this great utility tool.  I tried to fetch contest info for some users with `user_contest_info` API, but couldn't make it work.

Here's the code I run:
```ts
import { LeetCode } from "leetcode-query";

const leetcode = new LeetCode();
const info = await leetcode.user_contest_info("uwi");
```

I couldn't get any info but empty result. Is the API still working? or anything I can do to make it work.

**Comment by JacobLinCool** [comment]
Hello @corey-lin,

I've published a new version (`1.2.3`), which may fix the problem.

**Comment by corey-lin** [comment]
It works now with the new version 1.2.3. thanks!
